modules/nf-core/hifiasm/main.nf

```diff
@@ -41,6 +44,21 @@ process HIFIASM {
             -2 $maternal_kmer_dump \\
             $reads
 
+        cat <<-END_VERSIONS > versions.yml
+        "${task.process}":
+            hifiasm: \$(hifiasm --version 2>&1)
+        END_VERSIONS
+        """
+    } else if (use_hic) {
```

Comment:
In principle, can the HiC mode be used at the same time as the parental-kmer mode ?

Reply:
I don't think it can. I checked their paper and they don't mention the combination of both (but they don't say it's impossible neither). I think it would fail because it would try to do the phasing with two different technic. 
Maybe if it's something that can be done we can add the combined option later.

Reply:
OK ! Thanks for checking :+1: